packages/lisk-transactions/src/utils/validation/schema.ts

```diff
@@ -15,16 +15,7 @@
 export const transaction = {
 	$id: 'lisk/transaction',
 	type: 'object',
-	required: [
```

Comment:
I think asset should be removed as well from the required properties. Based on the schema definition in [lisk-core](https://github.com/LiskHQ/lisk/blob/development/framework/src/modules/chain/logic/transaction.js#L1433), it is not a required property.

Reply:
Done. Review again @SargeKhan please!